config/liota.conf

```diff
@@ -10,6 +10,7 @@ entity_file_path = /usr/lib/liota/entity
 iotcc_path = /usr/lib/liota/iotcc.json
 enable_reboot_getprop = False
 iotcc_load_retry = 3
+iotcc_response_timeout = 300
```

Comment:
Please add comment as to what the parameter does. Will be useful when someone wants to change it. Currently only liota developers understand these.

Reply:
Done